packit_service/service/urls.py

```diff
@@ -20,49 +20,47 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-from flask import url_for
+from packit_service.config import ServiceConfig
 
-from packit_service.service.app import packit_as_a_service as application
+DASHBOARD_URL = ServiceConfig().get_service_config().dashboard_url
 
 
-def get_srpm_log_url_from_flask(
+def _get_dashboard_url(build: str, id_: int) -> str:
+    """
+    Creates a dashboard URL that can be used for setting commit status or redirect.
+
+    Args:
+        build (str): Type of build, e.g. `srpm`, `copr` or `koji`.
+        id_ (int): Packit ID of the build.
+
+    Returns:
+        URL to the results of `id_` build of kind `build`.
+    """
+    return f"{DASHBOARD_URL}/results/{build}-builds/{id_}"
```

Comment:
What about `build_type`?

```suggestion
def _get_dashboard_url(build_type: str, id_: int) -> str:
    """
    Creates a dashboard URL that can be used for setting commit status or redirect.

    Args:
        build_type (str): Type of build, e.g. `srpm`, `copr` or `koji`.
        id_ (int): Packit ID of the build.

    Returns:
        URL to the results of `id_` build of kind `build`.
    """
    return f"{DASHBOARD_URL}/results/{build_type}-builds/{id_}"
```

Reply:
Yeah, I looked into testing farm, not sure if keep it, or generalize for TF too

Reply:
Good point. Since it's mostly the same thing, I would generalize for TF as well.